api/envoy/api/v2/core/config_source.proto

```diff
@@ -40,7 +41,8 @@ message ApiConfigSource {
   enum ApiType {
     // Ideally this would be 'reserved 0' but one can't reserve the default
     // value. Instead we throw an exception if this is ever used.
-    UNSUPPORTED_REST_LEGACY = 0 [deprecated = true];
+    UNSUPPORTED_REST_LEGACY = 0
```

Comment:
I'm kind of surprised this wasn't disabled earlier, we have zero support for this AFAIK.

Reply:
It was disabled, just the old way via the runtime strings :-)